app/site/_components/HeroContainerScroll.tsx

```
@@ -0,0 +1,196 @@
+"use client";
+import React from "react";
+import { useScroll, useTransform, motion, MotionValue } from "framer-motion";
+import Image from "next/image";
+import { TypewriterEffect } from "./TypewriterEffect";
+
+export const HeroContainerScroll = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)